cmd/nginx-ingress/main.go

```diff
@@ -80,16 +80,10 @@ var (
 	For Kubernetes < 1.18, the Ingress Controller only processes resources that belong to its class -
```

Comment:
Since support for K8s < 1.19 was also removed, we can make additional editing:

A class of the Ingress controller.
A corresponding IngressClass resource with the name equal to the class must be deployed. Otherwise,
	the Ingress Controller will fail to start.
The Ingress controller only processes resources that belong to its class - i.e. have the "ingressClassName" field resource equal to the class.
The Ingress Controller processes all the VirtualServer/VirtualServerRoute/TransportServer resources that do not have the "ingressClassName" field.

What do you think?

(the corresponding cli arg doc, helm values file and two docs will have to be updated as well)

Reply:
updated